integration_tests/features/Reorgs.feature

```diff
@@ -30,5 +30,62 @@ Feature: Reorgs
     And I mine a block on B at height 4 with an invalid MMR
     Then node B is at tip BTip1
 
-
-
+@reorg
+  Scenario: Pruned mode reorg past horizon
+    Given I have a base node NODE1 connected to all seed nodes    
+    When I mine a block on NODE1 with coinbase CB1
+    Given I have a base node NODE2 connected to node NODE1
+    When I mine but don't submit a block BLOCK2 on NODE2
+    When I mine but don't submit a block BLOCK3 on NODE2
+    When I mine but don't submit a block BLOCK4 on NODE2
+    When I mine but don't submit a block BLOCK5 on NODE2
+    When I mine but don't submit a block BLOCK6 on NODE2
+    When I mine but don't submit a block BLOCK7 on NODE2
+    When I mine but don't submit a block BLOCK8 on NODE2
+    When I mine but don't submit a block BLOCK9 on NODE2
+    When I mine but don't submit a block BLOCK10 on NODE2
+    When I mine but don't submit a block BLOCK11 on NODE2
+    When I mine but don't submit a block BLOCK12 on NODE2
+    When I mine but don't submit a block BLOCK13 on NODE2
+    When I mine but don't submit a block BLOCK14 on NODE2
+    When I mine but don't submit a block BLOCK15 on NODE2
+    When I mine but don't submit a block BLOCK16 on NODE2
+    When I mine but don't submit a block BLOCK17 on NODE2
+    When I mine but don't submit a block BLOCK18 on NODE2
+    When I mine but don't submit a block BLOCK19 on NODE2
+    When I mine but don't submit a block BLOCK20 on NODE2   
+    And I stop NODE2
+    When I mine 3 blocks on NODE1
+    When I create a transaction TX1 spending CB1 to UTX1
+    When I submit transaction TX1 to NODE1
+    Then NODE1 has TX1 in MEMPOOL state
+    When I mine 6 blocks on NODE1
+    Given I have a pruned node PNODE2 connected to node NODE1
+    Then node PNODE2 is at height 10
+    When I start NODE2
+    When I submit block BLOCK2 to NODE2
+    When I submit block BLOCK3 to NODE2
+    When I submit block BLOCK4 to NODE2
+    When I submit block BLOCK5 to NODE2
+    When I submit block BLOCK6 to NODE2
+    When I submit block BLOCK7 to NODE2
+    When I submit block BLOCK8 to NODE2
+    When I submit block BLOCK9 to NODE2
+    When I submit block BLOCK10 to NODE2
+    When I submit block BLOCK11 to NODE2
+    When I submit block BLOCK12 to NODE2
+    When I submit block BLOCK13 to NODE2
+    When I submit block BLOCK14 to NODE2
+    When I submit block BLOCK15 to NODE2
+    When I submit block BLOCK16 to NODE2
+    When I submit block BLOCK17 to NODE2
+    When I submit block BLOCK18 to NODE2
+    When I submit block BLOCK19 to NODE2
+    When I submit block BLOCK20 to NODE2
+    Then node NODE2 is at height 20
+    Then node NODE1 is at height 20
+    Then node PNODE2 is at height 20
+    # Because TX1 should have been re_orged out we should be able to spend CB1 again
+    When I create a transaction TX2 spending CB1 to UTX2
+    When I submit transaction TX2 to NODE1
+    Then NODE1 has TX2 in MEMPOOL state
```

Comment:
NODE1 is the wrong node to test here, you should be testing that the pruned node accepts it

Reply:
NODE1 and PNODE1 should have the same chain, butt swapping it over.